### PR TITLE
support for multiple listeners

### PR DESCRIPTION
**Pull request by kmdavis**

### CODE REVIEW

**Comment by kmdavis** [comment]
other broken lints are pre-existing

File: packages/core/src/receiver.ts [hunk 1]
```diff
@@ -115,10 +115,10 @@ export class RemoteReceiver {
         const text = this.attached.get(id) as RemoteTextSerialization;
         text.text = newText;
 
-        const listener = this.listeners.get(id);
+        const listeners = this.listeners.get(id);
 
-        if (listener) {
-          listener(text);
+        if (listeners) {
+          listeners.forEach((listener) => listener(text));
```
Comment (lemonmade) [inline packages/core/src/receiver.ts]: Please use `for of` here

File: packages/core/src/receiver.ts [hunk 2]
```diff
@@ -145,11 +152,13 @@ export class RemoteReceiver {
...
 
         for (const attached of queuedUpdates) {
-          const listener = this.listeners.get(
+          const listeners = this.listeners.get(
             attached === this.root ? ROOT_ID : attached.id,
           );
 
-          listener?.(attached);
+          if (listeners) {
+            listeners.forEach((listener) => listener(attached));
```
Comment (lemonmade) [inline packages/core/src/receiver.ts]: `for of` here too please

File: packages/core/src/receiver.ts [hunk 3]
```diff
@@ -31,7 +31,7 @@ export class RemoteReceiver {
 
   private readonly listeners = new Map<
     string | typeof ROOT_ID,
-    UpdateListener<any>
+    UpdateListener<any>[]
```
Comment (lemonmade) [inline packages/core/src/receiver.ts]: Please use a `Set`

**Review by lemonmade — APPROVED** [review]

**Comment by lemonmade** [comment]
New versions released with this included

**Pull request merged**